docs/using/vertex.md

```diff
@@ -8,7 +8,8 @@ Sentinel-1 RTC products are generated by ASF's HyP3 processing platform leveragi
 
 ## Getting Started
 
-To request RTC products, visit [ASF Data Search - Vertex](https://search.asf.alaska.edu).
+To request RTC products, visit [ASF Data Search - Vertex](https://search.asf.alaska.edu).  
+<img width="25" src="https://user-images.githubusercontent.com/25309951/96494192-7c4b4e80-11f2-11eb-9af9-4a54c0b449ca.png"> You will need to sign in on Vertex using your [Earthdata](https://urs.earthdata.nasa.gov/) credentials to submit your request.  
```

Comment:
I think inline is more consistant with the rest of this file

Reply:
@asjohnston-asf Can you weigh in? If you're not interested in incorporating this change, there's no reason to debate the placement of the icon, but I haven't heard one way or the other. 
Another option would be to include the Earthdata reference in the same sentence, but my guess was that you wanted to keep that first Vertex reference pretty clean. But it could read: 
To request RTC products, visit [Vertex] and sign in with your [Earthdata] credentials. (sign in logo here)